tools/smoke-tests/settings.gradle

```diff
@@ -18,27 +18,23 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-group 'io.spine.tools'
+include 'enrichment-lookup'
+include 'known-types'
+include 'validation-rules'
+include 'protoc-plugin'
+include 'external-known-types'
 
-apply from: deps.scripts.generateDescriptorSet
+final def links = [
+        'io.spine.tools:spine-model-compiler': ':model-compiler',
+        'io.spine:spine-base'                : ':base',
+        'io.spine:spine-base-testlib'        : ':testlib'
+]
 
-dependencies {
-    compile project(':base')
-
-    implementation deps.gen.javaPoet
-
-    testCompile project(':testlib')
-}
-
-protobuf {
-    generatedFilesBaseDir = generatedRootDir
-    protoc {
-        artifact = deps.build.protoc
-    }
-
-    generateProtoTasks {
-        all().each { final task ->
-            task.descriptorSetOptions.includeSourceInfo = true
+// :base root project
+includeBuild("$rootDir/../../") {
```

Comment:
Please describe the trick in details, so that an out-of-context person could understand and re-use for own purposes or education.